public/raw_articles/tba-bulk-data.md

```diff
@@ -0,0 +1,185 @@
+[//]: <author> (Andrew Schreiber)
+[//]: <title> (TBA API For Beginners)
```

Comment:
Change title to reflect BigQuery :) this is what gets rendered [on the article header](https://i.imgur.com/N3AIxGv.png) by abusing markdown "comments" (unrendered code) as metadata/headers.